Drivers/Src/CANbus.c

```diff
@@ -39,16 +39,29 @@ static void CANbus_CountIncoming(void) {
 /**
  * @brief   Initializes the CAN system
  * @param   loopback	: if we should use loopback mode (for testing)	
+ * @param 	faultState  : determines whether to implement Rx and Tx interrupts
```

Comment:
fix the whitespace

Reply:
i think this is just a github thing, I don't have that white space locally